packages/babel-parser/test/fixtures/estree/directives/function-strict-property-access/input.js

```diff
@@ -0,0 +1,5 @@
+
+function f() {
+  "use strict".foo;
```

Comment:
Can you move this rest inside `fixtures/core` instead of `fixtures/estree`?

Reply:
That makes sense. What is `fixtures/estree` for?

Reply:
It's for the parser plugin `estree`

Reply:
It's for the parser plugin `estree`